contrib/vexecutor/vexecutor.c

```diff
@@ -117,7 +122,7 @@ static void backportTupleDescriptor(PlanState* ps,TupleDesc td)
 		tuple = caql_getnext(pcqCtx);
 
 		if (!HeapTupleIsValid(tuple))
-			elog(ERROR, "cache lookup failed for type %u", oidtypeid);
+			continue;
```

Comment:
Why use `continue` instead of `elog ERROR` here?

Reply:
For the grouping aggregation, there may generate the tuple descriptor like this: {vtype,  vtype,  non-vtype, non-vtype}, we have to convert the vtype to non-vtype and skip the non-vtype which already exists.